x-pack/legacy/plugins/ml/server/routes/job_audit_messages.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { licensePreRoutingFactory } from '../new_platform/licence_check_pre_routing_factory';
+import { wrapError } from '../client/error_wrapper';
+import { RouteInitialization } from '../new_platform/plugin';
+import { jobAuditMessagesProvider } from '../models/job_audit_messages';
+
+/**
+ * Routes for job audit message routes
+ */
+export function jobAuditMessagesRoutes({ xpackMainPlugin, router }: RouteInitialization) {
+  /**
+   * @apiGroup JobAuditMessages
+   *
+   * @api {get} /api/ml/job_audit_messages/messages/:jobId Get audit messages
+   * @apiName GetJobAuditMessages
+   * @apiDescription Returns audit messages for specified job ID
+   */
+  router.get(
+    {
+      path: '/api/ml/job_audit_messages/messages/{jobId}',
+      validate: {
+        params: schema.object({ jobId: schema.maybe(schema.string()) }),
+        query: schema.maybe(schema.object({ from: schema.maybe(schema.any()) })),
+      },
+    },
+    licensePreRoutingFactory(xpackMainPlugin, async (context, request, response) => {
+      try {
+        const { getJobAuditMessages } = jobAuditMessagesProvider(
+          context.ml!.mlClient.callAsCurrentUser
+        );
+        const { jobId } = request.params;
+        const from = request.query.from;
```

Comment:
absolute nit, could be ` const { from } = request.query;`
it stands out be cause if the line above it

Reply:
646802be53d4ce19923a519f3591aff07b6cadb5